docs/v0.2.0/about_docs.rst

```diff
@@ -0,0 +1,9 @@
+About the documentation
```

Comment:
so now the `main` branch will contain `docs` dir per release (0.2.0, latest etc) ?

Reply:
Yes, like what envoy-website does too, this is how versioned docs work.

https://github.com/envoyproxy/envoy-website/tree/main/docs/envoy

Reply:
I provided a make target to make this run easily.